[T13-1]Tutor Pal

hrxiong opened the pull request.

xhsien reviewed (approved):
Good work :)

xhsien commented:
Good work. Some comments I have:

1. Please update the project page.
2. Update Travis badge on Readme.
3. Update User Guide Introduction section.

xhsien commented:
Also, add `coming in v2.0` for features not yet implemented in user guide.

xhsien commented:
Remember to add relevant diagrams (activity/class/object/sequence) in your developer guide to apply what we've learnt in class!

Commit:
further change

Commit:
further change

Commit:
further change

Commit:
further change

ccl17 committed:
Merge pull request #118 from junjiee/master

update commands for consistency and DeleteTimeCommand

ccl17 committed:
Merge pull request #120 from hrxiong/master

UserGuide.adoc: add more detail about grade attribute

Commit:
fix error

Commit:
change

LZYAndy committed:
Merge branch 'master' into fix-userguide-and-add-fuzzy-matching

ccl17 committed:
Merge remote-tracking branch 'upstream/master' into mybranch

junjiee committed:
Merge pull request #121 from junjiee/master

fix invalid addTime display error

Commit:
Merge commit 'e78373fa5b08d78106df4081c218f04f402ea330'

# Conflicts:
#	src/test/java/seedu/address/logic/commands/TimeAddCommandTest.java
#	src/test/java/systemtests/AddTimeCommandTest.java
#	src/test/java/systemtests/TimeAddCommandTest.java

Commit:
fix-userguide-and-add-fuzzy-matching

# Conflicts:
#	src/test/java/seedu/address/logic/commands/AddTimeCommandTest.java
#	src/test/java/systemtests/AddTimeCommandTest.java
#	src/test/java/systemtests/TimeAddCommandTest.java

ccl17 committed:
Edited ConfigTest for new app title

Commit:
further change

Commit:
further change

ccl17 committed:
Merge pull request #119 from chenlu-chua/mybranch

Edited SampleDataUtil to give sample persons timeslot

ccl17 committed:
Merge pull request #116 from LZYAndy/fix-userguide-and-add-fuzzy-matching

Fix userguide and add fuzzy matching

CJuanvip committed:
r

CJuanvip committed:
s

CJuanvip committed:
d

CJuanvip committed:
aa

CJuanvip committed:
e

CJuanvip committed:
d

CJuanvip committed:
s

CJuanvip committed:
sss

CJuanvip committed:
d

Commit:
change user guide

CJuanvip committed:
Merge branch 'master' into commandChange

hrxiong committed:
Merge pull request #122 from LZYAndy/change-user-guide

change find in user guide

CJuanvip committed:
d

CJuanvip committed:
Merge commit '3294ae23c38a18330cb9b3bf5b3bbeb02467df14' into commandChange

# Conflicts:
#	src/main/java/seedu/address/logic/parser/AddressBookParser.java

CJuanvip committed:
d

CJuanvip committed:
Merge branch 'commandChange' of https://github.com/CJuanvip/main into commandChange

# Conflicts:
#	src/main/java/seedu/address/logic/parser/AddressBookParser.java

CJuanvip committed:
Merge branch 'master' into commandChange

CJuanvip committed:
s

Commit:
further change

junjiee committed:
Merge pull request #123 from LZYAndy/change-user-guide

further change

ccl17 committed:
Updated user guide

ccl17 committed:
Updated EarningsCommand

ccl17 committed:
Fixed a bug in Edit command where education could not be edited

ccl17 committed:
Fixed Checkstyle

Commit:
further changes

Commit:
update time attribute in person

Commit:
update add parser, edit command and parser

Commit:
update model and storage

Commit:
correct model and storage

Commit:
further change

Commit:
update test

Commit:
Revert "update test"

This reverts commit 2e21b317aa84023842fae409abe6d5af3b77186b.

Commit:
Revert "Revert "update test""

This reverts commit 1138bdd59e3dc38f533e1510e7cdf4ce5209f103.

Commit:
Revert "Revert "Revert "update test"""

This reverts commit ddc4d9fa1545ee788aaf0437130c66b6da905ace.

Commit:
fix test

Commit:
fix style

Commit:
changed AddTimeCommand and DeleteTimeCommand to be by index instead of name

Commit:
fix test

Commit:
add DeleteTimeCommand, DeleteTimeCommandParser test, update UniquePersonListTest

Commit:
update UserGuide

Commit:
update test

Commit:
Merge remote-tracking branch 'upstream/master' into change-UI

CJuanvip committed:
Merge commit '423bd3e6f8475ad8bb33267e2178af6bc56bb55f' into commandChange

CJuanvip committed:
Update SuggestionCommandTest.java

CJuanvip committed:
Update SuggestionCommandParserTest.java

CJuanvip committed:
Update SuggestionCommandTest.java

CJuanvip committed:
Update SuggestionCommandTest.java

Commit:
Update the details about the grade attribute more properly.

ccl17 committed:
Implemented promote command

ccl17 committed:
Edited education

CJuanvip committed:
Update SuggestionCommandTest.java

ccl17 committed:
Removed unnecessary loggers

ccl17 committed:
Updated Ui.png

ccl17 committed:
Edited addcommandtest

ccl17 committed:
Merge pull request #113 from CJuanvip/commandChange

Command change

ccl17 committed:
Merge pull request #164 from junjiee/master

update time attribute, add and edit command

Commit:
change user guide

ccl17 committed:
Updated userguide

ccl17 committed:
Resolve conflicts

Commit:
change user guide

ccl17 committed:
Edited PromoteCommand

Commit:
Merge remote-tracking branch 'upstream/master'
merge from upstream/master

ccl17 committed:
Updated UserGuide with promote command details

ccl17 committed:
Updated promote commands

Commit:
Merge commit '353e0dfd01a868e67f6b5b806e20068f93f3f41d' into update-user-guide

# Conflicts:
#	docs/UserGuide.adoc
#	src/main/java/seedu/address/logic/commands/AddTimeCommand.java
#	src/main/java/seedu/address/logic/commands/DeleteTimeCommand.java

LZYAndy committed:
Merge pull request #165 from LZYAndy/change-UI

Change ui

LZYAndy committed:
Merge pull request #166 from hrxiong/master

Update the details about the grade attribute more properly.

LZYAndy committed:
Merge branch 'master' into update-user-guide

ccl17 committed:
Merge pull request #167 from LZYAndy/update-user-guide

Update user guide

LZYAndy committed:
Update UserGuide.adoc

Commit:
change on ui

Commit:
change on ui

Commit:
further change

Commit:
further change

Commit:
further change

Commit:
Add the deleteGrade command

Commit:
fix checkstyle

hrxiong committed:
Merge branch 'master' into master

Commit:
add the deleteGrade into userguide

CJuanvip committed:
Merge pull request #170 from hrxiong/master

add deleteGrade command

Commit:
update Ui.png

hrxiong committed:
Merge pull request #172 from hrxiong/master

update Ui.png

Commit:
updated AddTimeCommand and DeleteTImeCommand

Commit:
further change

Commit:
further change

Commit:
Merge commit 'b790880e9f1c3bd72de7e8067e350258bcae4a24' into update-user-guide

hrxiong committed:
Merge pull request #174 from junjiee/master

update AddTimeCommand and DeleteTimeCommand

Commit:
Merge commit '6e27cc76db4936ad186e64862a76f07cb427ded2' into update-user-guide

ccl17 committed:
resolve conflict

ccl17 committed:
Merge remote-tracking branch 'upstream/master' into mybranch

Commit:
further change

ccl17 committed:
Edited EditCommand EditPersonDescriptor

junjiee committed:
Merge pull request #163 from chenlu-chua/mybranch

 Fixed a bug in Edit command where education could not be edited

Commit:
Merge commit '840b2562c64996377665a0d4d3cc8e295e1e9181' into update-user-guide

Commit:
further change

Commit:
further change

Commit:
further change

Commit:
further change

ccl17 committed:
Merge remote-tracking branch 'upstream/master' into mybranch

Commit:
further change

ccl17 committed:
Fixed bug with promotecommand and updated equals and has of Person class

ccl17 committed:
Fixed checkstyle

Commit:
further change

ccl17 committed:
Edited EarningsCommand equsls

Commit:
further change

LZYAndy committed:
Merge pull request #175 from chenlu-chua/mybranch

Fixed bug for promotecommand

ccl17 committed:
Merge pull request #169 from LZYAndy/update-user-guide

Change UI

ccl17 committed:
Merge remote-tracking branch 'upstream/master' into mybranch

ccl17 committed:
Fixed a bug where tutorpal cant be reopened after added a time before 1000hrs

ccl17 committed:
Changed Graduated Tag to red color

ccl17 committed:
Fixed bug where number of students promoted shown is wrong

ccl17 committed:
Fixed checkstyle

ccl17 committed:
Added PPP

ccl17 committed:
Updated user and developer guide

Commit:
add test case for student panel

Commit:
Merge commit 'e709537f17d3070ba9da81a69903c14e77daea8e' into add-test-case-for-student-panel

Commit:
delete duplicate method

junjiee committed:
Merge pull request #177 from chenlu-chua/mybranch

 Fixed a bug where tutorpal cant be reopened after added a time before 1000 hrs

junjiee committed:
Merge pull request #178 from LZYAndy/add-test-case-for-student-panel

add test case for student panel

ccl17 committed:
Added PromoteIndividualCommand test cases

ccl17 committed:
Edited promote command

ccl17 committed:
Added test cases to PromoteIndividualCommandTest

ccl17 committed:
Merge remote-tracking branch 'upstream/master' into mybranch

ccl17 committed:
Added test cases for PromoteAllCommand

ccl17 committed:
Removed unused imports

ccl17 committed:
Fixed checkstyle

ccl17 committed:
Merge pull request #179 from chenlu-chua/mybranch

Added test cases for promote feature

ccl17 committed:
updated portfolio

CJuanvip committed:
Merge branch 'addTest'

Commit:
update ui picture to latest version

ccl17 committed:
Merge pull request #180 from LZYAndy/change-ui-picture-to-latest-version

update ui picture to latest version

Commit:
update Time attribute

CJuanvip committed:
edit

CJuanvip committed:
Merge branch 'master' of https://github.com/CJuanvip/main

# Conflicts:
#	src/main/java/seedu/address/logic/commands/FilterByEducationCommand.java
#	src/main/java/seedu/address/logic/commands/FilterByGradeCommand.java
#	src/main/java/seedu/address/logic/parser/FilterByEducationCommandParser.java
#	src/main/java/seedu/address/logic/parser/FilterByTimeCommandParser.java

Commit:
update UserGuide

CJuanvip committed:
Update SuggestionCommandByIndexParser.java

CJuanvip committed:
Update SuggestionCommandByIndex.java

CJuanvip committed:
Update ExchangeTimeCommandParser.java

CJuanvip committed:
Update SuggestionCommandByIndexParser.java

CJuanvip committed:
Update SuggestionCommandByIndex.java

CJuanvip committed:
Update SuggestionCommandByIndex.java

CJuanvip committed:
d

CJuanvip committed:
Merge branch 'master' of https://github.com/CJuanvip/main

# Conflicts:
#	src/main/java/seedu/address/logic/parser/ExchangeTimeCommandParser.java

ccl17 committed:
Updated user guide

Commit:
Add test for add command system test to test more details about grade attribute

CJuanvip committed:
d

CJuanvip committed:
Update ParserUtilTest.java

CJuanvip committed:
Update ExchangeTimeCommandTest.java

CJuanvip committed:
Update FilterByTimeCommandParserTest.java

LZYAndy committed:
Merge pull request #181 from CJuanvip/master

edit error

Commit:
Update DeveloperGuide for Time and AddTimeCommand

CJuanvip committed:
d

CJuanvip committed:
Merge branch 'master' of https://github.com/CJuanvip/main

CJuanvip committed:
d

Commit:
add test for edit command system test to test more details about grade attribute

CJuanvip committed:
Update SuggestionCommandTest.java

Commit:
fix checkstyle

Commit:
update DeveloperGuide

CJuanvip committed:
Update SuggestionCommandTest.java

Commit:
update developerguide

CJuanvip committed:
Update FilterByTimeCommandTest.java

LZYAndy committed:
Merge pull request #182 from CJuanvip/master

edit error

LZYAndy committed:
Merge pull request #183 from hrxiong/master

add test case for grade

Commit:
add DeleteGradeSystemTest.java

Commit:
tag userguide

Commit:
tag userguide

Commit:
add DeleteGradeCommandTest.java

Commit:
Add DeleteGradesCommandParserTest.java

Commit:
fix checkstyle

Commit:
add PPP

Commit:
Merge remote-tracking branch 'upstream/master'
merge into master

Commit:
update url in ppp

CJuanvip committed:
edit

CJuanvip committed:
Merge branches 'master' and 'master' of https://github.com/CJuanvip/main

CJuanvip committed:
Merge pull request #184 from hrxiong/master

add test for deleteGrade command

LZYAndy committed:
Merge pull request #185 from CJuanvip/master

edit

ccl17 committed:
Updated earnings command

ccl17 committed:
Edited userguide and developer guide

hrxiong committed:
Merge pull request #187 from chenlu-chua/mybranch

Updated earnings command

Commit:
add ppp and change developer guide

Commit:
fix checkstyle

Commit:
change some user guide, add PPP

hrxiong committed:
Merge branch 'master' into master

hrxiong committed:
Merge pull request #188 from LZYAndy/add-ppp-and-developer-guide

add ppp and change developer guide

hrxiong committed:
Merge branch 'master' into master

junjiee committed:
Merge branch 'master' into master

CJuanvip committed:
d

junjiee committed:
Merge pull request #189 from hrxiong/master

change some user guide, add PPP

junjiee committed:
Merge branch 'master' into master

CJuanvip committed:
error

Commit:
fix the use case in developer guide

ccl17 committed:
Updated readme

LZYAndy committed:
Merge pull request #186 from junjiee/master

update UserGuide, DeveloperGuide, PPP

LZYAndy committed:
Merge pull request #190 from CJuanvip/master

ppp some error

Commit:
fix conflict

LZYAndy committed:
Update lzyandy.adoc

CJuanvip committed:
ee

junjiee committed:
Merge pull request #191 from junjiee/master

fix conflict

LZYAndy committed:
Update lzyandy.adoc

ccl17 committed:
Edited ppp

junjiee committed:
Merge pull request #192 from CJuanvip/master

ee

Commit:
update aboutus

LZYAndy committed:
Update lzyandy.adoc

ccl17 committed:
Merge remote-tracking branch 'upstream/master' into mybranch

LZYAndy committed:
Update AboutUs.adoc

Commit:
update ppp and DG

junjiee committed:
Merge pull request #193 from junjiee/master

update aboutus

CJuanvip committed:
no message

ccl17 committed:
Merge remote-tracking branch 'upstream/master' into mybranch

ccl17 committed:
Updated aboutus

ccl17 committed:
Updated developerguide

ccl17 committed:
Merge pull request #194 from CJuanvip/master

no message

junjiee committed:
Merge pull request #195 from chenlu-chua/mybranch

updated documentation

Commit:
update PPP and developerGuide

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
update PPP

LZYAndy committed:
Update AboutUs.adoc

junjiee committed:
Merge pull request #196 from hrxiong/master

update PPP and developerGuide

ccl17 committed:
Updated mainapp version number

Commit:
update PPP

CJuanvip committed:
j

LZYAndy committed:
Merge pull request #197 from chenlu-chua/mybranch

v1.4

junjiee committed:
Merge pull request #198 from hrxiong/master

update PPP

ccl17 committed:
Merge pull request #199 from CJuanvip/master

ppp edit

ccl17 committed:
Update userguide

Commit:
update developerguide

junjiee committed:
Merge pull request #200 from chenlu-chua/mybranch

Update userguide

junjiee committed:
Merge pull request #201 from junjiee/master

update developerguide

Commit:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-1/main

Commit:
update developerguide

junjiee committed:
Merge branch 'master' into master

junjiee committed:
Merge pull request #202 from junjiee/master

update developerGuide
